rhobs/alerting/data_plane/prometheus.exporter_alerts.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: konflux-exporter-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: exporter_alerts
+    interval: 5m
+    rules:
+    - alert: KonfluxUpAlert
+      expr: absent_over_time(konflux_up{namespace="appstudio-grafana-datasource-exporter"}[5m]) == 1
```

Comment:
I think we should also add the `check` label to the expression here, so if we add another konflux_up metric on that namespace in the future then we won't get false negatives.

Reply:
Can we use `absent` instead of `absent_over_time`?
Using queries that look at a range vector (e.g. `[5m]`) makes it less trivial to understand when they are supposed to alert, comparing to metrics looking at instant vector (no `[]`).